src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -60,12 +60,17 @@ $active-poster-shadows: 0 10px 20px 0 rgba($primary, 0.56);
     font-size: $font-size--md;
     font-weight: $font-weight--600;
 
-    margin: $spacing--md 0;
+    margin: $spacing--sm 0;
   }
 
   &__categories {
     display: flex;
     flex-wrap: wrap;
+    margin: $spacing--sm 0;
+
+    -webkit-line-clamp: 2;
+    line-clamp: 2;
+    overflow: hidden;
```

Comment:
What do we need this for?

Reply:
There are cases where some posters in the exhibitors hall have a dozen categories — this prevents them from overflowing the posterpreview card, but still allows them to be searchable.